src/app/rpmostree-pkg-builtins.cxx

```diff
@@ -110,7 +112,7 @@ pkg_change (RpmOstreeCommandInvocation *invocation,
 
   /* Use newer D-Bus API only if we have to. */
   g_autofree char *transaction_address = NULL;
-  if (met_local_pkg)
+  if (met_local_pkg || opt_apply_live)
```

Comment:
The basic API *should* work for this since it's through the `options` arg.

Reply:
True, but I didn't want to explicitly commit to supporting the new option in the old DBus interface.  You're right it likely works though.